ios/browser/api/query_filter/utils.h

```diff
@@ -8,6 +8,8 @@
 
 #import <Foundation/Foundation.h>
 
+#include <optional>
+
```

Comment:
Please remove, we cannot have C++ headers in public Obj-C headers

Reply:
Done.